drivers/hmis/spec/requests/hmis/organization_spec.rb

```diff
@@ -75,5 +75,19 @@
       orgs = result.dig('data', 'organizations', 'nodes')
       expect(orgs.length).to eq(0)
     end
+
+    context 'when there are multiple HMIS data sources' do
+      let!(:ds2) { create :hmis_data_source }
+      let!(:o5) { create :hmis_hud_organization, data_source: ds2, user: u1, OrganizationName: 'Pineapples' }
+      let!(:ds2_access_control) { create_access_control(hmis_user, ds2) }
+
+      it 'does not return organizations from other data source' do
```

Comment:
This is the spec test that reproduces the maybe-bug I wrote about above https://github.com/greenriver/hmis-warehouse/pull/4788/files#r1786621775

Reply:
Does this reproduce it? I can't tell from the assertion, since its just saying there are 4 nodes.

Reply:
In fact it didn't, because the model factories were set up wrong. It turns out that 
```
let!(:o5) { create :hmis_hud_organization, data_source: ds2, user: u1 }
```
results in an o5 that is still in ds1.

```
let!(:o5) { create :hmis_hud_organization, data_source: ds2 }
```
is correct. No bug here!

Reply:
:| right, because u1 overwrites the data source id